docs/conceptual/copy-and-update-record-expression-[fsharp].md

```diff
@@ -0,0 +1,40 @@
+---
+title: Copy and Update Record Expression (F#)
+description: Copy and Update Record Expression (F#)
+keywords: visual f#, f#, functional programming
+author: ChrSteinert
+manager: danielfe
+ms.date: 06/04/2016
+ms.topic: language-reference
+ms.prod: visual-studio-dev14
+ms.assetid: b5fc4ef0-64eb-4272-96a7-bb4dffbb634a
+---
+
+# Copy and Update Record Expression (F#)
+
+A *copy and update record expression* is an expression that copies an existing record, updates specified fields and returns the updated record.
+
+
+## Syntax
+
+``` fsharp
+{ record-variable with
```

Comment:
nit: `record-name` would be preferable to `record-variable` here, since that's the terminology used in things like `let` bindings.